### ISSUE TITLE
[pulseaudio-ogon] support for multi audio format

### ISSUE THREAD

**Issue by amazingfate** [ogon-project/ogon-project#18]
As I see in [code](https://github.com/ogon-project/pulseaudio-ogon/blob/master/src/modules/ogon/ogon-sink.c#L417), pulseaudio-ogon only support WAVE_FORMAT_PCM, which will cost a lot of bandwith. An encoded format such as WAVE_FORMAT_DVI_PCM(supported by both windows7 and windows10) or WAVE_FORMAT_AAC_MS(supported by windows10) will improve the audio/video sync performance when playing some 1080p video on some low-end platform.

**Comment by akallabeth** [comment]
True, FreeRDP `freerdp_dsp_supports_format` and `freerdp_dsp_encode` can be utilized for transcoding.
